src/week3/UniqueTriplets.java

```diff
@@ -1,9 +1,56 @@
 package week3;
 
+import java.util.Arrays;
+//program to find unique triplets adding to a given number 
+//taking into consideration an unsorted array
 public class UniqueTriplets {
 
 	public static void main(String[] args) {
-		
-	}
+		int array[]={-1,0,2,1,-1,1,5};
+		int sum=0;
+		int result[][]=tripletArrayBruteForce(array,sum);
+		System.out.print("[");
+		for(int i=0;i<result.length;i++){
+			System.out.print("[");
+			for(int j=0;j<result[i].length;j++){
+				if(j==result[i].length-1){	
+					System.out.print(result[i][j]);
+				}
+				else System.out.print(result[i][j]+",");
+			}
+			if(i==result.length)
+				System.out.print("]");
+			else
+				System.out.print("],");
 
+		}
+		System.out.print("]");
+	}
+	private static int[][] tripletArrayBruteForce(int[] array,int sum){
+		int returnArray[][]=new int[array.length][3];

```

Comment:
Yes, agree . I have used this in the optimized approach and since it was brute force method i left it to be.